index.html

```diff
@@ -99,14 +99,14 @@ <h2 id="faq">F.A.Q.</h2>
     <h2 id="basic">Basic Usage</h2>
 
     <p>On first start a new game is created and you can choose your edition to start a new campaign. The current game
-      is saved in the browser, so no game state is lost
-      when closing the page and coming back later (please note: private/inkognito browsing modes of course deletes
+      is saved in the browser, so no game data is lost
+      when you browse elsewhere and come back later (please note: private/incognito browsing modes will of course delete
       this
       data on closing!).</p>
 
     <p>Most setup is done via the <strong>Main Menu <img class="icon"
-          src="https://gloomhaven-secretariat.de/assets/images/menu.svg"></strong> on the top left
-      corner. A shortcut for current action is located next to the menu icon.</p>
```

Comment:
There's a server icon and a text description of the turn state at the top of the page. I couldn't figure out what this "shortcut for current action" text referred to so I removed it because it didn't seem accurate/important. Please put it back and clarify if I'm missing what it refers to!

Reply:
![image](https://github.com/Lurkars/ghs-help/assets/2760570/609aecb6-65db-4ab8-b989-073379f4a9c9)

The shortcuts are the texts hints next to the menu.

Reply:
Ahhh, after you're in a scenario there is just text. I'll change the wording to encompass what might be there more generally.